modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/portlet/action/test/EditSegmentsEntryMVCRenderCommandTest.java

```diff
@@ -64,14 +67,15 @@ public class EditSegmentsEntryMVCRenderCommandTest {
 			new LiferayIntegrationTestRule(),
 			PermissionCheckerMethodTestRule.INSTANCE);
 
-	@BeforeClass
-	public static void setUpClass() throws Exception {
-		_company = _companyLocalService.fetchCompany(
-			TestPropsValues.getCompanyId());
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		_company = _companyLocalService.getCompany(_group.getCompanyId());
```

Comment:
Keep the company in the setupClass so we don't need to run the query every time a test is executed.

Because the company won't in every test execution, as the group does.

Reply:
Oh yes, I forgot it :(,I totally agree with you, It is not necessary to get the company in every test execution.

Reply:
@cgoncas done!